projects/www/src/app/pages/guide/signals/signal-method.md

```diff
@@ -1,6 +1,6 @@
 # SignalMethod
 
-`signalMethod` is a standalone factory function used for managing side effects with Angular signals. It accepts a callback and returns a processor function that can handle either a static value or a signal. The input type can be specified using a generic type argument:
+`signalMethod` is a standalone factory function used for managing side effects with Angular signals. It accepts a callback and returns a processor function that can handle either a static value or a reactive computation, i.e. either a Signal or a function, which will be tracked automatically. The input type can be specified using a generic type argument:
```

Comment:
To avoid explaining multiple times that the Signal is compatible with a computation function, wdyt about mentioning a computation function in addition to Signal? (the same comment for other sections where this was changed)

```suggestion
`signalMethod` is a standalone factory function used for managing side effects with Angular signals. It accepts a callback and returns a processor function that can handle either a static value, a signal, or a computation function. The input type can be specified using a generic type argument:
```

Reply:
Do you mean I should also remove the example then? Otherwise, I don't see so many places where the docs become easier.
Should we really call it "computation function"? I used "reactive computation", because that's the offical Angular term: https://angular.dev/api/core/linkedSignal.

We could just say "reactive computation" without explanation, but I doubt that most people know what that is...